Metalama.Framework/Advising/IAdviser.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) SharpCrafters s.r.o. See the LICENSE.md file in the root directory of this repository root for details.
+
+using JetBrains.Annotations;
+using Metalama.Framework.Aspects;
+using Metalama.Framework.Code;
+using Metalama.Framework.Validation;
+
+namespace Metalama.Framework.Advising;
+
+[InternalImplement]
+[CompileTime]
+[PublicAPI]
+public interface IAdviser<out T>
```

Comment:
I think this interface and its members should be documented.

Reply:
I have added a task,.